speckle/base/resource.py

```diff
@@ -125,11 +139,42 @@ def _parse_response(self, response, comment=False, schema=None):
     def make_request(self, method, path, data=None, comment=False, schema=None):
         r = self._prep_request(method, path, comment, data)
         resp = self.s.send(r)
+        if not resp.ok:
```

Comment:
What error is this catching? What status codes are we expecting back that we would want to ignore and not warn the user about? I'm not sure how a user would know that they received a 403 not authorised response for example.

Reply:
This is to catch a 403 Unauthorized error. Parts of the Blender code are able to call `client.streams.list()` or whatever without being logged in, or if the authentication is expired. In that case, there is no response body, so `make_request()` throws an error.

Reply:
Right I get you. Is it worth throwing a requests error instead of using a silent print + return None? We could use `requests` [raise for status](https://requests.readthedocs.io/en/master/api/#requests.Response.raise_for_status) object method. In this case you would replace the `if not resp.ok:` line + loop below with `resp.raise_for_status()`. That way we get a meaningful error and dependent code can handle it however it pleases.

Reply:
This now skips the printing and just returns `Response.status_code`.

Reply:
(sorry forgot to refresh and see your replies)

That sounds better than just returning the status code. Will make the change.